#!/usr/bin/env python
PACKAGE = "stereo_vo"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("num_features", int_t, 0, "Number of features to detect", 100, 10, 200)
gen.add("min_features", int_t, 0, "Minimum number of features to track", 50, 10, 100)
gen.add("max_level", int_t, 0, "0-based maximal pyramid level number", 3, 0, 5)
gen.add("win_size", int_t, 0, "Size of the search window at each pyramid level", 11, 1, 21)

exit(gen.generate(PACKAGE, "stereo_vo", "StereoVoDyn"))
